content/zh/docs/tasks/observability/metrics/classify-metrics/index.md

```diff
@@ -24,7 +24,11 @@ Istio 允许您使用 AttributeGen 插件创建分类规则，该插件将请求
 您可以将属性用作 Istio 标准指标中的维度。
 相似地，您可以基于 `ListReviews` 和 `CreateReviews` 这类其他操作跟踪指标。
 
-## 按请求分类指标{#classify-metrics-by-request}
+Istio 使用 Envoy 代理生成指标，并在
+[manifests/charts/istio-control/istio-discovery/templates/telemetryv2_{{< istio_version >}}.yaml]({{<github_blob>}}/manifests/charts/istio-control/istio-discovery/templates/telemetryv2_{{< istio_version >}}.yaml)
+提供 `EnvoyFilter` 的相关配置。因此，编写分类规则需要向 `EnvoyFilter` 添加属性。
```

Comment:
Is anything I missed? I cannot find this para in en upstream: https://github.com/istio/istio.io/blob/master/content/en/docs/tasks/observability/metrics/classify-metrics/index.md